canonical-web-and-design/jaas-dashboard
# Sort models by status and display them in separate tables

## Done

Naively sort the models by status and then display them in separate tables. 

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- If you have models with units who's agents are `lost` then you'll see them in the topmost table.

## Details

Fixes https://github.com/canonical-web-and-design/juju-squad/issues/929

We don't have a spec for this so at the moment it's only splitting the models by applications that have units who's agents are `lost`. This will need a follow-up to enhance this sorting functionality but it's good to get the component output landed so we can style things.

I'll leave https://github.com/canonical-web-and-design/juju-squad/issues/930 open to track for the follow-up

## Screenshots

For those who may not have models in this state:

<img width="1337" alt="Screen Shot 2019-10-16 at 2 49 09 PM" src="https://user-images.githubusercontent.com/532033/66958175-dedc9e00-f024-11e9-96c4-92b042da1509.png">
